### PR TITLE
Update AAD Sample

### PR DESCRIPTION
**Pull request by Netyyyy**
This quick update needs to merge before Sample release.

### CODE REVIEW

File: aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md [hunk 1]
```diff
@@ -97,52 +97,35 @@ spring:
...
-4. (Optional) If you want to configure oauth2 redirect uri, please configure by :
-```properties
-redirectUriTemplate: xxxxxx-your-redirect-uri-xxxxxx
-```
-
 ---
- ### Step 4: Change Role_group1 to your group
-1. You can use `@PreAuthorize` annotation or `UserPrincipal` to manage access to web API based on user's group membership. You will need to change `ROLE_group1` to groups you want to allow to access the API in `TodoListController.java` or you will get "Access is denied".
+ ### Step 4: Create Role_group1 to your group
+1. In the left-hand navigation pane, select **Groups**, and then select **New group** to create a new group named `ROLE_group1`, choose your tenant as **Owners** and Spring Boot Sample as **Members**.
```
Comment (saragluna) [inline aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md:120]: this is not correct, the goup name should be group1?

File: aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md [hunk 2]
```diff
@@ -97,52 +97,35 @@ spring:
...
    
 ---
 
 ### Step 5: Give it a run
 
 * Run with Maven 
  ```
- cd azure-spring-boot-samples/aad/azure-spring-boot-starter-active-directory/aad-resource-server-by-filter
+ cd azure-spring-boot-samples/aad/spring-cloud-azure-starter-active-directory\aad-resource-server-by-filter
```
Comment (saragluna) [inline aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md:128]: cd azure-spring-boot-samples/aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter

**Review by saragluna — CHANGES_REQUESTED** [review]

File: aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter-stateless/README.md [hunk 3]
```diff
@@ -32,27 +32,30 @@ For the test SPA provided with this example you should create the following role
       "allowedMemberTypes": [
         "User"
       ],
+      "description": "Full admin access",
       "displayName": "Admin",
-      "id": "2fa848d0-8054-4e11-8c73-7af5f1171001",
+      "id": "********-****-****-****-************",
```
Comment (backwind1233) [inline aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter-stateless/README.md:37]: I suggest you keep the original value;
the value is just UUID as I know, if the user just want to follow the readme to set up the environment,
it's better for users to just copy and paste.

File: aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md [hunk 4]
```diff
@@ -97,52 +97,35 @@ spring:
...
+      enabled: true
+      credential:
+        client-id: ${AZURE_CLIENT_ID}
+        client-secret: ${AZURE_CLIENT_SECRET}
+      profile:
+        tenant-id: ${AZURE_TENANT_ID}
+      user-group:
+        allowed-group-names: group1,group2
+      redirect-uri-template: http://localhost:8080/
+      jwt-connect-timeout: 5000
```
Comment (saragluna) [inline aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md]: wrong indent

File: aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md [hunk 5]
```diff
@@ -103,46 +103,29 @@ spring:
           client-secret: ${AZURE_CLIENT_SECRET}
         profile:
           tenant-id: ${AZURE_TENANT_ID}
-        redirect-uri-template: http://localhost:8080/
-        # groups that you created in your Azure AD tenant
         user-group:
           allowed-group-names: group1,group2
-        # Optional, the default value is 
-        # environment: global  
+        redirect-uri-template: http://localhost:8080/
```
Comment (saragluna) [inline aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md:108]: we don't need this I think?

**Review by saragluna — APPROVED** [review]
LGTM.

File: aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md (hunk 5 above)
Comment (Netyyyy) [reply aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md:108]: It's already configured in application.yml and checking the result needs this.

**Pull request merged**